ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/BonsaiWorldStateUpdater.java

```diff
@@ -47,20 +47,30 @@
 public class BonsaiWorldStateUpdater extends AbstractWorldUpdater<BonsaiWorldView, BonsaiAccount>
     implements BonsaiWorldView {
 
-  private final Map<Address, BonsaiValue<BonsaiAccount>> accountsToUpdate = new HashMap<>();
-  private final Map<Address, BonsaiValue<Bytes>> codeToUpdate = new HashMap<>();
-  private final Set<Address> storageToClear = new HashSet<>();
+  private Map<Address, BonsaiValue<BonsaiAccount>> accountsToUpdate = new HashMap<>();
+  private Map<Address, BonsaiValue<Bytes>> codeToUpdate = new HashMap<>();
+  private Set<Address> storageToClear = new HashSet<>();
```

Comment:
non-blocking feedback:  we could have a private constructor that takes these params, and keep them final.  We'd have to add a matching protected constructor in AbstractWorldStateUpdater too though.